cmd/kava/cmd/app.go

```diff
@@ -62,7 +64,7 @@ func (ac appCreator) newApp(
 
 	homeDir := cast.ToString(appOpts.Get(flags.FlagHome))
 	snapshotDir := filepath.Join(homeDir, "data", "snapshots") // TODO can these directory names be imported from somewhere?
-	snapshotDB, err := sdk.NewLevelDB("metadata", snapshotDir)
+	snapshotDB, err := cometbftdb.NewGoLevelDB("metadata", snapshotDir)
```

Comment:
This is a regression here -- should be `cometbftdb.NewDB("metadata", server.GetAppDBBackedn(appOpts), snapshotDir)` to open using configured db type